ProcessLib/UncoupledProcessesTimeLoop.h

```diff
@@ -110,6 +112,16 @@ class UncoupledProcessesTimeLoop
      */
     bool solveCoupledEquationSystemsByStaggeredScheme(
         const double t, const double dt, const std::size_t timestep_id);
+
+    /**
+     *  Find the minimum time step size among the predicted step sizes of
+     *  processes and step it as common time step size.
+     *
+     *  @param t           Current time
+     */
+    double computeTimeStepping(const double prev_dt, double& t,
```

Comment:
This seems to be used only in the implementation. Is it necessary to have it in the interface?